src/main/java/fi/metatavu/dcfb/server/search/index/IndexableLocation.java

```diff
@@ -0,0 +1,207 @@
+package fi.metatavu.dcfb.server.search.index;
+
+import java.time.OffsetDateTime;
+import java.util.List;
+import java.util.UUID;
+
+import fi.metatavu.dcfb.server.search.annotations.Field;
+
+/**
+ * Indexable implementation for locations
+ */
+public class IndexableLocation extends AbstractIndexable {
+
+  public static final String TYPE = "location";
+  public static final String CREATED_AT_FIELD = "createdAt";
+  public static final String MODIFIED_AT_FIELD = "modifiedAt";
+  public static final String SLUG_FIELD = "slug";
+  
+  @Field(analyzer = "finnish")
+  private List<String> nameFi;
+
+  @Field(analyzer = "swedish")
+  private List<String> nameSv;
+
+  @Field(analyzer = "english")
+  private List<String> nameEn;
+  
+  @Field(analyzer = "finnish")
+  private List<String> additionalInformationsFi;
+
+  @Field(analyzer = "swedish")
+  private List<String> additionalInformationsSv;
+
+  @Field(analyzer = "english")
+  private List<String> additionalInformationsEn;
+
+  @Field(index = "not_analyzed", store = true)
+  private String slug;
+
+  @Field(index = "not_analyzed", store = true, type = "date")
+  private OffsetDateTime createdAt;
+
+  @Field(index = "not_analyzed", store = true, type = "date")
+  private OffsetDateTime modifiedAt;
+
+  public IndexableLocation() {
+    // Zero-argument constructor
+  }
+
+  /**
+   * Constructor
+   * 
+   * @param id id
+   * @param nameFi name in finnish
+   * @param nameSv name in swedish
+   * @param nameEn name in english
+   * @param slug slug
+   * @param createdAt created at
+   * @param modifiedAt modified at
+   */
+  public IndexableLocation(UUID id, List<String> nameFi, List<String> nameSv, List<String> nameEn, 
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Constructor has 10 parameters, which is greater than 7 authorized. ([squid:S00107](https://sonarcloud.io/organizations/metatavu-oy/rules#rule_key=squid%3AS00107))

[See it in SonarCloud](https://sonarcloud.io/project/issues?id=fi.metatavu.dcfb-api%3Adcfb-api&pullRequest=11&issues=AWQt_biLYwZU9vvrs3K4&open=AWQt_biLYwZU9vvrs3K4)

[ISSUE_KEY]: AWQt_biLYwZU9vvrs3K4